tests/integration/errors/test_oapi_uncaught_handled_exception_04.php835.php

```diff
@@ -0,0 +1,176 @@
+<?php
+/*
+ * Copyright 2020 New Relic Corporation. All rights reserved.
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+/*DESCRIPTION
+When a user exception handler unregisters itself as an exception handler when it handles uncaught exception,
+the agent should record the error and add error attributes on all spans leading to uncaught exception as
+well as the one throwing the exception. Error attributtes are not expected on the root span (because
+the exception has been handled) as well as on the span created for exception handler.
+*/
+
+/*INI
+newrelic.distributed_tracing_enabled=1
+newrelic.transaction_tracer.threshold = 0
+newrelic.span_events_enabled=1
+newrelic.code_level_metrics.enabled = 0
+display_errors=1
+log_errors=0
+*/
+
+
+/*SKIPIF
+<?php
+if (version_compare(PHP_VERSION, "8.3", "<")) {
```

Comment:
test is named php835, but this just checks for 8.3?

Reply:
The 835 naming convention was perfectly fine, but the 835 test only had this check:

```
<?php
if (version_compare(PHP_VERSION, "8.3", "<")) {
  die("skip: PHP < 8.3.0 not supported\n");
}
```

shouldn't it be 

```<?php
if (version_compare(PHP_VERSION, "8.3.5", "<")) {
  die("skip: PHP < 8.3.5 not supported\n");
}
```

Reply:
I assumed 835 meant php 8.3.5 and above.

Reply:
Maybe instead of in the name, update the description inside the test explaining why the skipif and which versions we expect it to run on.
Then you can have an 83a, 83b, etc to show different variants of the test for 8.3?

Reply:
Thanks for the suggestions. See 501325667d95cc2a820141ae8ffbd0168fb51bbe.